tests/README.md

```diff
@@ -74,6 +153,35 @@ Container-based tests:
 - `--docker`: Uses Docker as container engine. Accepts the base image name.
 - `--container`: Uses Podman as container engine. Accepts the base image name.
 
+To use container backends, use the following options on the command line:
+
+- Podman - `--container <base image name>`
+- Docker - `--docker <base image name>`
+
+This is an example to run test scenarios for rule `rule_sshd_disable_kerb_auth`.
+Note that you need to have root privileges for both cases:
+
+Using Podman:
+
+```
+./test_suite.py rule --container ssg_test_suite --datastream ../build/ssg-centos7-ds.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml rule_sshd_disable_kerb_auth
+```
```

Comment:
This is an example of rule-based testing. I suggest moving this example to section `Rule-based testing`.